test/Interop/SwiftToCxx/functions/cdecl.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) %s -typecheck -module-name CdeclFunctions -emit-cxx-header-path %t/empty.h
+// RUN: %FileCheck %s < %t/empty.h
+
+// RUN: %check-cxx-header-in-clang -std=c++14 %t/empty.h
+// RUN: %check-cxx-header-in-clang -std=c++17 %t/empty.h
+
+// CHECK-LABEL: namespace CdeclFunctions {
+
+// CHECK: namespace _impl {
+// CHECK: extern "C" int cfuncPassTwo(int x, int y) noexcept;
+// CHECK: }
+
+@_cdecl("cfuncPassTwo")
```

Comment:
It's kind of interesting that this cdecl name is different from the declared fn, so maybe the name should be `differentCDeclName` or something like that. What do you think?

(Or maybe the test name?)

Reply:
Yeah, that makes sense to me.